classes/openxt-image-disk.bbclass

```diff
@@ -0,0 +1,110 @@
+# Portions from openembedded-core/meta/syslinux.bbclass
+# Copyright (C) 2004-2006, Advanced Micro Devices, Inc.  All Rights Reserved
+# Released under the MIT license (see packages/COPYING)
+#
+# Portions from openembedded-core/meta/image-vm.bbclass
+# (loosly based off image-live.bbclass Copyright (C) 2004, Advanced Micro
+# Devices, Inc.)
+
+# The "disk" conversion creates a syslinux bootable disk with a small FAT
+# partition and a second rootfs partition.  It is basically a CONVERSION_CMD
+# implmentation of image-vm & syslinux bbclasses inlined together, but limited
+# in functionality for the limited use case.
+
+BOOTDD_EXTRA_SPACE ?= "2048"
+KERNEL_CMDLINE     ?= "root=/dev/xvda2 ro console=hvc0"
+
+CONVERSIONTYPES_append = " disk"
+
+CONVERSION_DEPENDS_disk = "syslinux \
+                           syslinux-native \
+			   dosfstools-native \
+			   virtual/kernel \
+			   parted-native \
+			   mtools-native \
+			   "
+
+CONVERSION_CMD_disk() {
+	BOOTDIR="${S}/bootp/boot"
+	BOOTIMG="${S}/bootp.image"
+	ROOTIMG=${IMAGE_NAME}${IMAGE_NAME_SUFFIX}.${type}
+	IMAGE=$ROOTIMG.disk
+
+	#populate_kernel $BOOTDIR
```

Comment:
Is this a comment or code that's commented out? This one and the syslinux one below seem unclear to me...

Reply:
That's fine, it just looks odd to those who are not familiar with the original code.